x/ecocredit/core/genesis_test.go

```diff
@@ -1,173 +1,163 @@
 package core_test
 
 import (
+	"context"
 	"encoding/json"
 	"testing"
 
+	"github.com/cosmos/cosmos-sdk/orm/model/ormdb"
+	"github.com/cosmos/cosmos-sdk/orm/model/ormtable"
+	"github.com/cosmos/cosmos-sdk/orm/testing/ormtest"
+	"github.com/cosmos/cosmos-sdk/orm/types/ormjson"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
 	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/types/known/timestamppb"
 
+	api "github.com/regen-network/regen-ledger/api/regen/ecocredit/v1"
+	"github.com/regen-network/regen-ledger/x/ecocredit"
 	"github.com/regen-network/regen-ledger/x/ecocredit/core"
 )
 
 func TestValidateGenesis(t *testing.T) {
-	x := `{
-		"regen.ecocredit.v1.BatchBalance":[
-			{
-				"batch_key":"1",
-				"address":"gydQIvR2RUi0N1RJnmgOLVSkcd4=",
-				"tradable":"90.003",
-				"retired":"9.997"
-			}
-		],
-		"regen.ecocredit.v1.Batch":[
-			{
-				"issuer":"WCBEyNFP/N5RoS4h43AqkjC6zA8=",
-				"project_key":"1",
-				"denom":"BIO01-00000000-00000000-001",
-				"start_date":"2021-04-08T10:40:10.774108556Z",
-				"end_date":"2022-04-08T10:40:10.774108556Z"
-			},
-		    {
-				"issuer":"gydQIvR2RUi0N1RJnmgOLVSkcd4=",
-				"project_key":"1",
-				"denom":"BIO02-00000000-00000000-001",
-				"start_date":"2021-04-08T10:40:10.774108556Z",
-				"end_date":"2022-04-08T10:40:10.774108556Z"
-			}
-		],
-		"regen.ecocredit.v1.BatchSupply":[
-			{
-				"batch_key":"1",
-				"tradable_amount":"90.003",
-				"retired_amount":"9.997"
-			}
-		],
-		"regen.ecocredit.v1.Class":[
-			{
-				"id":"BIO001",
-				"admin":"4A/V6LMEL2lZv9PZnkWSIDQzZM4=",
-				"credit_type_abbrev":"BIO"
-			},
-		    {
-				"id":"BIO02",
-				"admin":"HK9YDsBMN1hU8tjfLTNy+qjbqLE=",
-				"credit_type_abbrev":"BIO"
-			}
-		],
-		"regen.ecocredit.v1.Project":[
-			{
-				"id":"P01",
-				"admin":"gPFuHL7Hn+uVYD6XOR00du3C/Xg=",
-				"class_key":"1",
-				"jurisdiction":"AQ"
-			},
-			{
-				"id":"P02",
-				"admin":"CHkV2Tv6A7RXPJYTivVklbxXWP8=",
-				"class_key":"2",
-				"jurisdiction":"AQ",
-				"metadata":"project metadata"
-			}
-		],
-		"regen.ecocredit.v1.CreditType": [
-			{
-				"name":"carbon",
-				"abbreviation": "C",
-				"unit":"metric ton C02 equivalent",
-				"precision":6
-			},
-			{
-				"name":"biodiversity",
-				"abbreviation": "BIO",
-				"unit":"acres",
-				"precision":6
-			}
-		]
-	}`
+	ormCtx := ormtable.WrapContextDefault(ormtest.NewMemoryBackend())
+	modDB, err := ormdb.NewModuleDB(&ecocredit.ModuleSchema, ormdb.ModuleDBOptions{})
+	require.NoError(t, err)
+	ss, err := api.NewStateStore(modDB)
+
+	require.NoError(t, ss.CreditTypeTable().Insert(ormCtx, &api.CreditType{
+		Abbreviation: "BIO",
+		Name:         "biodiversity",
+		Unit:         "acres",
+		Precision:    6,
+	}))
+
+	require.NoError(t, ss.BatchBalanceTable().Insert(ormCtx,
+		&api.BatchBalance{BatchKey: 1, Address: sdk.AccAddress("addr1"), Tradable: "90.003", Retired: "9.997"}))
```

Comment:
> It would be nice to make these all a little easier to read as you have done elsewhere.

Maybe this was a bit unclear but I was hoping that we could make all of these a little easier to read and only made a suggestion for one as an example. There are other places where this small improvement could be made.

```suggestion
		&api.BatchBalance{
			BatchKey: 1,
			Address: sdk.AccAddress("addr1"),
			Tradable: "90.003",
			Retired: "9.997",
		}))
```